apps/desktop/desktop/src/main/resources/css/text.css

```
@@ -773,4 +773,24 @@
+
+.transparent-text-field .material-text-field-bg,
+.transparent-text-field .material-text-field-bg-hover {
+    -fx-background-color: -bisq-dark-grey-20;
+}
+
+.transparent-text-field .material-text-field-description-small {
+    -fx-font-size: 0.9em;
+}
+
+.transparent-text-field .material-text-field {
+    -fx-font-size: 1.1em;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)